test/stablecoin_converter.js

```diff
@@ -199,8 +199,9 @@ contract("StablecoinConverter", async (accounts) => {
       const orderId = [orderId1, orderId2]
       const tokenIdsForPrice = basicTrade.solution.tokenIdsForPrice
 
+      // Note that we claim a better solution than we have...
       await truffleAssert.reverts(
-        stablecoinConverter.submitSolution(batchIndex, owner, orderId, [0, 0], prices, tokenIdsForPrice, { from: solutionSubmitter }),
+        stablecoinConverter.submitSolution(batchIndex, 1, owner, orderId, [0, 0], prices, tokenIdsForPrice, { from: solutionSubmitter }),
```

Comment:
I would prefer to calculate the claimed values.

Then refactoring later the test is much better.

Reply:
Yes. You will notice that this is a todo in the Draft PR description.